ghost/core/core/server/services/themes/validate.js

```
@@ -1,5 +1,6 @@
 const debug = require('@tryghost/debug')('themes');
 const _ = require('lodash');
+const path = require('path');
```

ESLint

[error] 'path' is assigned a value but never used.

(no-unused-vars)